CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
 
 ## vNEXT
+- Seize sponsor on success task. (#79)
 - Refund sponsor on `claim`. (#77)
 - Sponsor match orders boost. (#67, #78)
+- Start migration to hardhat tests:
+    - `finalize` (#79)
 - Migrate to hardhat tests related to:
     - `initialize` (#74, #75)
```

Comment:
```suggestion
- Migrate to hardhat tests related to:
    - `finalize` (#79)
    - `initialize` (#74, #75)
```

Reply:
?

Reply:
Yes but migration is not over that's why.. When done could be moved